app/sidekiq/benefits_intake_status_job.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class BenefitsIntakeStatusJob
+  include Sidekiq::Job
+
+  def perform
+    pending_form_submission_ids = FormSubmission
+                                  .joins(:form_submission_attempts)
+                                  .where('form_submission_attempt.aasm_state = pending')
+                                  .map(&:benefits_intake_uuid)
+    response = BenefitsIntakeService::Service.get_bulk_status_of_uploads(pending_form_submission_ids)
```

Comment:
I don't love design pattern name spacing.  I would call this something like `BenefitsIntakeService::StatusPoller`

Reply:
NM, i see this is existing code : (

Reply:
Yeah I'm with you; don't love the naming. But I thought it belonged in the `Service` class since that's where other API calls live.